core/src/main/scala/kafka/log/Log.scala

```diff
@@ -266,10 +266,6 @@ class Log(val dir: File,
       activeSegment.timeIndex.resize(config.maxIndexSize)
     }
 
-    /**
-     * Load the largest timestamp into memory.
-     */
-    logSegments.foreach(_.loadLargestTimestamp())
```

Comment:
Is removing the above code correct? It doesn't seem there is code to compute the largest timestamp lazily.